flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/FlinkKubeClient.java

```diff
@@ -104,6 +107,67 @@ KubernetesWatch watchPodsAndDoCallback(
 		Map<String, String> labels,
 		WatchCallbackHandler<KubernetesPod> podCallbackHandler);
 
+	/**
+	 * Create the ConfigMap with specified content. If the ConfigMap already exists, nothing will happen.
+	 *
+	 * @param configMap ConfigMap.
+	 *
+	 * @return Return the ConfigMap create future.
+	 */
+	CompletableFuture<Void> createConfigMap(KubernetesConfigMap configMap);
```

Comment:
After a consideration, I will change the behavior of `CreateConfigMap`. If it is already existing, a `FlinkRuntimeExeception` will be thrown. Because the existing ConfigMap may be created externally and will make the Flink cluster works not as expected.

So the name will still be `createConfigMap`.

Reply:
Makes sense to me.